### ISSUE TITLE
Package subpath './lib' is not defined by "exports"

### ISSUE THREAD

**Issue by aplr** [mswjs/interceptors#285]
I just ran `npm update`, which updated `headers-polyfill` to the just released `3.1.0`, which modifies the exports in `package.json`. This is the error I get:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib' is not defined by "exports" in /myapp/node_modules/headers-polyfill/package.json
    at new NodeError (node:internal/errors:387:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:464:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:748:3)
    at resolveExports (node:internal/modules/cjs/loader:493:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:533:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:942:27)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/myapp/node_modules/@mswjs/interceptors/lib/interceptors/ClientRequest/NodeClientRequest.js:89:13) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

Issue is `import { Headers, objectToHeaders } from 'headers-polyfill/lib'` in `NodeClientRequest.ts` ([here](https://github.com/mswjs/interceptors/blob/94458729283528fe8977ccfdf04257ec5ea4f053/src/interceptors/ClientRequest/NodeClientRequest.ts#L5)), as `lib` is not exposed any more.

Guess the fix is
```diff
- import { Headers, objectToHeaders } from 'headers-polyfill/lib'
+ import { Headers, objectToHeaders } from 'headers-polyfill'
```

**Comment by jordevo** [comment]
👋  having this same issue here since last release `3.1.0`... temp fix for us was setting `headers-polyfill` on our project to previous release `3.0.10`.

**Comment by kettanaito** [comment]
Opened a fix in #286

**Comment by jordevo** [comment]
thank you @kettanaito !

had this quick *PR* there just in case... 
https://github.com/mswjs/interceptors/pull/287/files

I see you're also upgrading the package version...

**Comment by kettanaito** [comment]
## Released: v0.17.5 🎉

This has been released in v0.17.5!

- 📄 [**Release notes**](https://github.com/mswjs/interceptors/releases/tag/v0.17.5)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/interceptors/v/0.17.5)

Make sure to always update to the latest version (`npm i @mswjs/interceptors@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
